2024/2024-ssl-black-white-hulls-rules.adoc

```diff
@@ -0,0 +1,50 @@
+:source-highlighter: highlightjs
+
+= RoboCup 2024 SSL Black and White Hulls Technical Challenge Rules
+{docdate}
+:toc:
+:sectnumlevels: 0
+
+// add icons from fontawesome in a up-to-date version
+ifdef::backend-html5[]
+++++
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+++++
+endif::backend-html5[]
+
+:icons: font
+:numbered:
+
+NOTE: References to the male gender in the rules with respect to referees, team
+members, officials, etc. are for simplification and apply to both males and
+females.
+
+== Goals of the Technical Challenge
+
+Given that most parts of the teams use similar hulls, differing only in the use of stickers and sponsorships, it can be difficult for spectators to differentiate the teams during a match only by observing the central blob color. Because of it, the TC/OC decided that, in the future, it would be preferable that opponent teams use different hull colors in a match. One team would use black hulls and the other team white hulls. So, the main goal of the challenge is that the teams bring ideas that solve this efficiently and don't make logistics too complicated.
+
+=== Participation Requirements
+
+All teams are eligible and *are encouraged* to participate in this challenge.
+
+== Evaluation
+
+Every team has a 5-minute pitch to explain their solution for the challenge and the built design and 5 minutes for possible questions. After the presentation, the TC/OC will measure the time necessary to switch from a black hull to a white hull of one robot.
+
+At least one member of each team should be present during presentations since all teams will vote for the best solution.
+
+The following criteria will be evaluated.
+
+* Size efficiency
+* Durability
+* Ease of changing
```

Comment:
Should 'costs' be a factor, too? If a team for example build a circular shaped lcd display as their hull that can display all colors with the press of a button, it might have impressive switching time of a few us, and it might be very size efficient and maybe even durable, but nothing we could ask all teams to bring, no?

Reply:
Maybe we could add an 'Ease of building and costs' factor, that includes the costs and how easy is to construct the solution? Like, in RoboFEI we use carbon fiber hulls, but they are kind difficult to make. Or we could split this factor in two, 'ease of building' and 'costs'.